dronegen/mac.go

```diff
@@ -365,19 +465,59 @@ func darwinTagBuildCommands(b buildType, opts darwinBuildOptions) []string {
 		`make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes`,
 	)
 
-	if b.hasTeleportConnect() {
+	// TODO: Remove building Connect and Node stuff from tag pipeline. However, these are currently
+	// shared between push and tag. Push will have to continue building Connect without signing.
+	if opts.hasTeleportConnect {
 		commands = append(commands,
 			`cd $WORKSPACE_DIR/go/src/github.com/gravitational/webapps`,
 			// c.extraMetadata.version overwrites the version property from package.json to $VERSION
 			// https://www.electron.build/configuration/configuration.html#Configuration-extraMetadata
-			`yarn install --frozen-lockfile && yarn build-term && yarn package-term -c.extraMetadata.version=$VERSION`,
```

Comment:
Removed because it has no effect due to how the webapps repo is set up, see gravitational/webapps#1047.